### PR TITLE
2022-10-11-Mango-Markets.md

### PR DESCRIPTION
**Pull request by jwalin12**

### CODE REVIEW

**Comment by jwalin12** [comment]
Wallet 1 placed a MNGO-SOL Perp Sell Order for 483 million MNGO perpetual futures at 3.8 cents per future ❌ , 
Wallet 2 bought the MNGO-SOL Sell Order from Wallet 1, effectively opening a MNGO long position for Wallet 2. ❌ and
Hacker's Wallet 1 and Wallet 2 were funded with 5 million USDC each. ❌ can be verified by the markdown links already attached

File: content/attacks/posts/2022-10-11-Mango-Markets.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+On October 11th, 2022, Mango Markets, a decentralized exchange on Solana was exploited. The hacker manipulated the price oracle for the protocol's MNGO token. The hacker first took out a long MNGO position on Mango. Then the hacker artificially raised the price of MNGO token by advantage of low liquidity on secondaries. The hacker then used the temporary high price of MNGO to take out loans of USDC, various other stable coins, and SOL against unrealized profit on the long MNGO position. 
+
+## Attacker
+
+The Attacker revealed himself in a series of [tweets](https://twitter.com/avi_eisen/status/1581326197241180160) under the pseudonym of Avraham Eisenberg.
+
+
+## Losses
+
+About USD $116 Million worth of crypto assets in terms of USDC, stablecoins and SOL was borrowed and never returned to the platform.
```
Comment (JediFaust) [inline content/attacks/posts/2022-10-11-Mango-Markets.md:18]: Funds were recovered by the exploiter, add a source

**Review by JediFaust — CHANGES_REQUESTED** [review]

File: content/attacks/posts/2022-10-11-Mango-Markets.md (hunk 1 above)
Comment (jwalin12) [reply content/attacks/posts/2022-10-11-Mango-Markets.md]: Updated

**Review by JediFaust — CHANGES_REQUESTED** [review]
Please, make sure you are checking the article using [language tool service](https://languagetool.org/) and fix grammar and punctuation issues. Spelling issues are okay, since it detects token symbols as a mistake but the are not. Also, read [new changes to headers](https://github.com/1712n/dn-institute#readme) and apply them.

File: content/attacks/posts/2022-10-11-Mango-Markets.md [hunk 2]
```diff
@@ -1,33 +1,36 @@
 ---
 date: 2022-10-11
+target-entities: Mango Markets
+entity-types: Protocol
```
Comment (JediFaust) [inline content/attacks/posts/2022-10-11-Mango-Markets.md:4]: Rewrite as follows, and discover [new changes](https://github.com/1712n/dn-institute/tree/main#readme) with list of common examples
```code
entity-types:
  - DeFi
  - Exchange

File: content/attacks/posts/2022-10-11-Mango-Markets.md [hunk 3]
```diff
@@ -1,33 +1,36 @@
...
+attack-types: Protocol Vulnerability, Oracle Exploit
```
Comment (JediFaust) [inline content/attacks/posts/2022-10-11-Mango-Markets.md:5]: Rewrite as:
```code
attack-types:
  - Price Oracle Manipulation

**Review by JediFaust — CHANGES_REQUESTED** [review]
Please, merge [main](https://github.com/1712n/dn-institute/tree/main) into the branch

**Review by JediFaust — CHANGES_REQUESTED** [review]

File: content/attacks/posts/2022-10-11-Mango-Markets.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+- DeFi
+- Exchange
+attack-types: 
+- Price Oracle Manipulation
+title: "Mango Markets Exploited for $116 Million"
+---
+
+## Summary
+
+On October 11th, 2022, Mango Markets, a decentralized exchange on Solana, was exploited. The hacker manipulated the price oracle for the protocol's MNGO token. The hacker first took out a long MNGO position on Mango. Then the hacker artificially raised the price of the MNGO token by taking advantage of low liquidity on secondary markets. The hacker then used the temporary high price of MNGO to take out loans of USDC, various other stable coins, and SOL against unrealized profit on the long MNGO position.
```
Comment (JediFaust) [inline content/attacks/posts/2022-10-11-Mango-Markets.md:15]: Please, rewrite the summary, with less `the hacker` word. Rephrase or combine sentences, and use synonyms as `attacker` etc.

**Review by JediFaust — CHANGES_REQUESTED** [review]

**Review by JediFaust — APPROVED** [review]
Thanks! Make sure to check grammar issues in further contributions)

**Comment by albina-at-inca** [comment]
@jwalin12 can you please email your BTC address to [albina.bashaeva@inca.digital](mailto:albina.bashaeva@inca.digital)?

Please include the QR code.

**Pull request merged**